internal/Restore-DBFromFilteredArray.ps1

```diff
@@ -371,7 +390,7 @@ Function Restore-DBFromFilteredArray
 					$RestoreDirectory = ((Split-Path $Restore.RelocateFiles.PhysicalFileName) | sort-Object -unique) -join ','
 					$RestoredFile = (Split-Path $Restore.RelocateFiles.PhysicalFileName -Leaf) -join ','
 				}
-				if (!$ScriptOnly)
+				if ($ScriptOnly -eq $false)
```

Comment:
No idea. But it was part of the issue with Copy-DbaDatabase not cleaning up as well.